source/includes/extracts-watch-option.yaml

```diff
@@ -1,14 +1,14 @@
 ref: watch-option-batchSize
 content: |
-  Specifies the batch size for the cursor, which will apply to both the initial
-  ``aggregate`` command and any subsequent ``getMore`` commands. This determines
-  the maximum number of change events to return in each response from the
+  The maximum number of documents within each batch returned in a query result, which applies 
+  to the ``aggregate`` command. By default, the ``aggregate`` command has an initial batch size of 
+  ``101`` documents and a maximum size of 16 mebibytes for each subsequent batch. This
+  option can enforce a smaller limit than 16 mebibytes, but not a larger
+  one. If you set ``batchSize`` to a limit that results in batches larger than
+  16 MiB, this option has no effect.
+
+  This command determines the maximum number of change events to return in each response from the
   server.
-
-  Irrespective of the ``batchSize`` option, the initial ``aggregate`` command
-  response for a change stream generally does not include any documents
-  unless another option is used to configure its starting point (e.g.
-  ``startAfter``).
```

Comment:
I didn't notice this earlier, but this paragraph seems like a relevant bit of information to preserve.